Run flutter format --set-exit-if-changed .
Formatted packages/smooth_app/lib/pages/user_management/login_page.dart
Formatted 334 files (1 changed) in 1.53 seconds.
Formatting failed: 1
Error: Process completed with exit code 1.

--- a/packages/smooth_app/lib/pages/user_management/login_page.dart
+++ b/packages/smooth_app/lib/pages/user_management/login_page.dart
@@ -35,7 +35,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
 
   Future<void> _login(BuildContext context) async {
     final UserManagementProvider userManagementProvider =
-    context.read<UserManagementProvider>();
+        context.read<UserManagementProvider>();
     if (!_formKey.currentState!.validate()) {
       return;
     }
@@ -253,7 +253,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
                             ),
                           ),
                           shape:
-                          MaterialStateProperty.all<RoundedRectangleBorder>(
+                              MaterialStateProperty.all<RoundedRectangleBorder>(
                             const RoundedRectangleBorder(
                               borderRadius: CIRCULAR_BORDER_RADIUS,
                             ),
@@ -264,7 +264,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
                             context,
                             MaterialPageRoute<void>(
                               builder: (BuildContext context) =>
-                              const ForgotPasswordPage(),
+                                  const ForgotPasswordPage(),
                             ),
                           );
                         },
@@ -292,7 +292,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
                               context,
                               MaterialPageRoute<bool>(
                                 builder: (BuildContext context) =>
-                                const SignUpPage(),
+                                    const SignUpPage(),
                               ),
                             );
                             if (registered == true) {
@@ -319,7 +319,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
                           ),
                           child: Padding(
                             padding:
-                            const EdgeInsetsDirectional.only(bottom: 2.0),
+                                const EdgeInsetsDirectional.only(bottom: 2.0),
                             child: Text(
                               appLocalizations.create_account,
                               style: theme.textTheme.bodyText2?.copyWith(
@@ -350,7 +350,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
         context: context,
         builder: (BuildContext context) {
           final AppLocalizations appLocalizations =
-          AppLocalizations.of(context);
+              AppLocalizations.of(context);
 
           return SmoothAlertDialog(
             body: Text(appLocalizations.app_rating_dialog_title_enjoying_app),
@@ -371,7 +371,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
           context: context,
           builder: (BuildContext context) {
             final AppLocalizations appLocalizations =
-            AppLocalizations.of(context);
+                AppLocalizations.of(context);
 
             return SmoothAlertDialog(
               body: Text(
@@ -394,11 +394,11 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
       }
       bool? userRatedApp;
       if (enjoyingApp != null && enjoyingApp) {
-        userRatedApp =await showDialog<bool>(
+        userRatedApp = await showDialog<bool>(
           context: context,
           builder: (BuildContext context) {
             final AppLocalizations appLocalizations =
-            AppLocalizations.of(context);
+                AppLocalizations.of(context);
 
             return SmoothAlertDialog(
               body: Text(appLocalizations.app_rating_dialog_title),
@@ -416,7 +416,7 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
           },
         );
       }
-      if(userRatedApp!=null && userRatedApp) {
+      if (userRatedApp != null && userRatedApp) {
         await preferences.markInAppReviewAsShown();
       }
     }

